cscs-checks/tools/profiling_and_debugging/perftools.py

```diff
@@ -22,48 +22,51 @@ def __init__(self, lang, **kwargs):
             # FIXME: PGI Fortran is hanging after completion
             self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu']
 
+        # NOTE: Reduce time limit because for PrgEnv-pgi even if the output
+        # is correct, the batch job uses all the time.
```

Comment:
Should we open a ticket to ops for that?

Reply:
JG also suggested to open a ticket earlier today. (yet, I could imagine he meant the tools-rt and not immediately ops)